swagger_parser/test/generator/data_classes_test.dart

```diff
@@ -2759,4 +2759,43 @@ data class ClassName(
       expect(filledContent.content, expectedContent);
     });
   });
+
+  test('int64 to bigint ', () async {
+    const dataClass = UniversalComponentClass(
+      name: 'ClassName',
+      imports: {},
+      description: 'Test class',
+      parameters: [
+        UniversalType(type: 'integer', format: 'int64', isRequired: false),
```

Comment:
Plz add name for a type to avoid the name "null"

Reply:
Done! :)